### ISSUE TITLE
Does the pytorch NFSP work?

### ISSUE THREAD

**Issue by VitamintK** [deepmind/open_spiel#1004]
There are examples of the [tensorflow NFSP](https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/algorithms/nfsp.py): https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/examples/kuhn_nfsp.py and https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/examples/leduc_nfsp.py

I ran these and saw that the average exploitability decreases.

I couldn't find any examples of the [pytorch NFSP](https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/pytorch/nfsp.py) other than [the test](https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/pytorch/nfsp_pytorch_test.py) which doesn't check performance. I modified the `python/examples/kuhn_nfsp.py` example to use the pytorch NFSP instead, but the exploitability shoots up to 1. Is there a test or example showing that the pytorch NFSP works, and I'm doing something wrong? Or is it possible that there's a bug that prevents it from working?

**Comment by VitamintK** [comment]
hm, it looks like my test with the pytorch NFSP is improving, albeit very slowly: it shoots up to 0.99 exploitability during the first 20,000 episodes, but then slowly decreases (down to 0.96 at 440,000 episodes). This is slower than the tensorflow example which goes down to 0.15 exploitability after 50,000 episodes, but it's clearly improving. Will look into it and continue updating here.

**Comment by VitamintK** [comment]
ok yeah it works fine, I just made a mistake when I replaced the tensorflow NFSP with the pytorch NFSP in the example code (I forgot to keep `**kwargs` in the arguments).

It would still be good to add an option to `python/examples/kuhn_nfsp.py` and `python/examples/leduc_nfsp.py` to use pytorch NFSP instead of TF, or add a separate example for pytorch NFSP (although it would be 99% the same). That would be a separate issue though, so I'll close this.
